yarn-project/simulator/src/public/public_processor.ts

```diff
@@ -135,6 +139,10 @@ export class PublicProcessor implements Traceable {
       }
     }
 
+    const duration = timer.s();
+    const rate = duration > 0 ? totalGas.l2Gas / duration : 0;
```

Comment:
Non blocking: I think it would be cool to calculate the rate in Grafana (e.g. adding to a counter the gas used after each tx)

Reply:
I don't think that would work would it? I think I actually want histograms here.